cmd/resolve.go

```diff
@@ -32,18 +35,35 @@ func NewResolveCmd() *cobra.Command {
 				}
 			}
 
+			if resolvehelperopts.baseSystem == "scratch" {
+				resolvehelperopts.baseSystem = ""
+			}
+
 			install, forceIgnored, err := resolve(repos, required)
 			if err != nil {
 				return err
 			}
 
-			if err := template.Render(os.Stdout, install, forceIgnored); err != nil {
-				return err
+			if !resolveopts.jsonOutput {
+				return template.Render(os.Stdout, install, forceIgnored)
```

Comment:
I'm not making use of this feature anymore, shall we drop it?

Reply:
Sure!